src/main/java/com/mraof/minestuck/data/tag/MinestuckItemTagsProvider.java

```diff
@@ -97,18 +103,34 @@ protected void addTags(HolderLookup.Provider provider)
 		copy(MSTags.Blocks.EMERALD_ORES, MSTags.Items.EMERALD_ORES);
 		copy(MSTags.Blocks.CRUXITE_STORAGE_BLOCKS, CRUXITE_STORAGE_BLOCKS);
 		
-		tag(SWORDS).add(EMERALD_SWORD.get(), SORD.get(), PAPER_SWORD.get(), CACTACEAE_CUTLASS.get(), STEAK_SWORD.get(), BEEF_SWORD.get(), IRRADIATED_STEAK_SWORD.get(), MACUAHUITL.get(), FROSTY_MACUAHUITL.get(), KATANA.get(), UNBREAKABLE_KATANA.get(), ANGEL_APOCALYPSE.get(), FIRE_POKER.get(), TOO_HOT_TO_HANDLE.get(), CALEDSCRATCH.get(), CALEDFWLCH.get(), ROYAL_DERINGER.get(), CLAYMORE.get(), CUTLASS_OF_ZILLYWAIR.get(), REGISWORD.get(), CRUEL_FATE_CRUCIBLE.get(), SCARLET_RIBBITAR.get(), DOGG_MACHETE.get(), COBALT_SABRE.get(), QUANTUM_SABRE.get(), SHATTER_BEACON.get(), SHATTER_BACON.get(), SUBTRACTSHUMIDIRE_ZOMORRODNEGATIVE.get(), MUSIC_SWORD.get(), PILLOW_TALK.get(), KRAKENS_EYE.get(), CINNAMON_SWORD.get());
-		tag(AXES).add(EMERALD_AXE.get(), BATLEACKS.get(), COPSE_CRUSHER.get(), QUENCH_CRUSHER.get(), MELONSBANE.get(), CROP_CHOP.get(), THE_LAST_STRAW.get(), BATTLEAXE.get(), CANDY_BATTLEAXE.get(), CHOCO_LOCO_WOODSPLITTER.get(), STEEL_EDGE_CANDYCUTTER.get(), BLACKSMITH_BANE.get(), REGIAXE.get(), GOTHY_AXE.get(), SURPRISE_AXE.get(), SHOCK_AXE.get(), SHOCK_AXE_UNPOWERED.get(), SCRAXE.get(), LORENTZ_DISTRANSFORMATIONER.get(), PISTON_POWERED_POGO_AXEHAMMER.get(), RUBY_CROAK.get(), HEPHAESTUS_LUMBERJACK.get(), FISSION_FOCUSED_FAULT_FELLER.get(), BISECTOR.get(), FINE_CHINA_AXE.get(),
-				LIPSTICK_CHAINSAW.get(), CAKESAW.get(), MAGENTA_MAULER.get(), THISTLEBLOWER.get(), EMERALD_IMMOLATOR.get(), OBSIDIATOR.get(), DEVILS_DELIGHT.get(), DEMONBANE_RAGRIPPER.get(), FROSTTOOTH.get());
-		tag(PICKAXES).add(EMERALD_PICKAXE.get(), MINE_AND_GRIST.get(), PROSPECTING_PICKSCYTHE.get());
-		tag(SHOVELS).add(EMERALD_SHOVEL.get(), WOODEN_SPOON.get(), SILVER_SPOON.get(), MELONBALLER.get(), TERRAIN_FLATENATOR.get(), NOSFERATU_SPOON.get(), THRONGLER.get(), CROCKER_SPOON.get());
+		List<MSToolType> customMiningToolTypes = List.of(SICKLE_TOOL, SCYTHE_TOOL, CLAWS_TOOL, HAMMER_TOOL, FORK_TOOL);
+		List<MSToolType> customMeleeToolTypes = List.of(LANCE_TOOL, CLUB_TOOL, KNIFE_TOOL, KEY_TOOL, FAN_TOOL, BATON_TOOL, STAFF_TOOL, CANE_TOOL, WAND_TOOL);
```

Comment:
It'd probably be good with a code comment describing what it means to add a tool type to these lists.